samples/headless-ssr/commerce-generic/src/server.ts

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+import {engineDefinition} from './common/engine.js';
+import {getNavigatorContext} from './common/navigatorContext.js';
+import {renderApp} from './common/renderApp.js';
+import {renderHtml} from './common/renderHtml.js';
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.static('dist'));
+
+app.get('/', async (req, res) => {
+  try {
+    engineDefinition.searchEngineDefinition.setNavigatorContextProvider(() =>
+      getNavigatorContext(req)
+    );
+
+    const staticState =
+      await engineDefinition.searchEngineDefinition.fetchStaticState({});
```

Comment:
Why is it an empty object ? Assuming its for brevity sake ?

Maybe could just showcase how a `searchEngineDefinition.fetchStaticState` would typically be used on a route with some query param (eg `?query=foo`) to filter the result set ?

Reply:
maybe this branch is not using the latest changes made on the ssr-next.
fetchStaticState should always at least define the context.
Do you remember from which branch you created this PR?